### PR TITLE
notif sys

### PR DESCRIPTION
**Pull request by vezcraz**
The post methods in the transfer_form class based views have been updated with the notif sys. Whenever the student submits the application form, an email will be sent to the hod's email id with all the details of the student.
This won't work properly now since the submit button is not functional.

### CODE REVIEW

**Comment by ChetanKhanna** [comment]
@Oytrik test this once and let us know because last I remember, the submit button was working fine. Atleast, the models were getting updated.

**Comment by Oytrik** [comment]
@ChetanKhanna It's working. Mail is getting sent.

File: transfers/views/student_views.py [hunk 1]
```diff
@@ -67,6 +68,24 @@ def post(self, request, *args, **kwargs):
...
+                    "\nName: " + data.applicant.user.first_name + " " +data.applicant.user.last_name +
+                    "\nTransfer Type: " + data.get_sub_type_display()+
+                    "\nCGPA: " + str(data.cgpa)+
+                    "\nThesis Locale: " + data.get_thesis_locale_display()+
+                    "\nThesis Subject: " + data.thesis_subject+
+                    "\nOrganization Name: " + data.name_of_org+
+                    "\nExpected Outcome: " + data.expected_deliverables)
+                print("done")
+                send_mail("Transfer Application: " + request.user.username, body,
+                    'psdmail@gmail.com',[str(data.hod_email)],
```
Comment (ChetanKhanna) [inline transfers/views/student_views.py]: This email is not the same as one defined in `settings.py`, why is it so?

File: transfers/views/student_views.py [hunk 2]
```diff
@@ -105,6 +124,21 @@ def post(self, request, *args, **kwargs):
...
+            if request.user == data.applicant.user:
+                body =  str("\nID: " + data.applicant.user.username+
+                    "\nName: " + data.applicant.user.first_name + " " +data.applicant.user.last_name +
+                    "\nTransfer Type: " + data.get_sub_type_display()+
+                    "\nCGPA: " + str(data.cgpa)+
+                    "\nReason For Transfer: " + data.reason_for_transfer+
+                    "\nOrganization Name: " + data.name_of_org)
+                print("done")
+                send_mail("Transfer Application: "  + request.user.username, body,
+                    'psdmail@gmail.com',[str(data.hod_email)],
```
Comment (ChetanKhanna) [inline transfers/views/student_views.py]: If possible, can you move this entire block of code into a new function in `utils.py`? And call that function from here. Also,is it really needed to iterate the `TS2PSTransfer` table (I am referring to `for data in TS2PSTransfer.objects.all()`)?

Same remarks apply to the above code blocok.

Comment (vezcraz) [reply transfers/views/student_views.py]: fixed

File: transfers/views/student_views.py (hunk 1 above)
Comment (vezcraz) [reply transfers/views/student_views.py]: i don't know why it's working this way. This was an unintended error. Also fixed.

**Pull request merged**